spec.html

```diff
@@ -124,7 +193,7 @@ <h1>Iterator ( )</h1>
         <h1>Value Properties of the Iterator Constructor</h1>
         <emu-clause id="sec-iterator.prototype">
           <h1>Iterator.prototype</h1>
-          <p>The initial value of Iterator.prototype is %Iterator.prototype%.</p>
+          <p>The initial value of Iterator.prototype is %IteratorPrototype%.</p>
```

Comment:
please revert this.

Reply:
This refers to the [`%IteratorPrototype%`](https://tc39.es/ecma262/#sec-%iteratorprototype%-object) intrinsic specified in <https://tc39.es/ecma262/#sec-well-known-intrinsic-objects>.

Reply:
Yes, and we're trying to get rid of %FooPrototype% intrinsics in favor of `%Foo.prototype%.